README.md

```diff
@@ -81,17 +81,17 @@ Acá importa para la clasificación cómo van apareciendo las clases y sus tiemp
 ```
 Nota: es de suma importancia los rangos de los timestamps. Pues si no se tiene en cuenta algun frame el mismo va a ser clasificado con clase None.
 
-1. Moverse a la carpeta `rcnn`.
-2. Reentrená la CNN usando el retrain dentro de rcnn. Corré el comando:
+2. Moverse a la carpeta `rcnn`.
+3. Reentrená la CNN usando el retrain dentro de rcnn. Corré el comando:
 ```
-python3 ./cnn/retrain.py --bottleneck_dir=./cnn/bottleneck --model_dir=./cnn/inception --output_graph=./cnn/retrained_graph.pb --output_labels=./cnn/retrained_labels.txt --image_dir ./test
+python3 ./cnn/retrain.py --bottleneck_dir=./cnn/bottleneck --model_dir=./cnn/inception --output_graph=./cnn/retrained_graph.pb --output_labels=./cnn/retrained_labels.txt --image_dir ./videos
```

Comment:
`./rcnn/retrain.py` or directly `retrain` if you are in the folder. And the same for the others.

Reply:
It specifically says to move to the rcnn folder before running the comand.